extras/userdoc/md/documentation/installation.md

```diff
@@ -184,49 +164,52 @@ the help pages. The browser is set as an option of `helpdesk`. Please see the
 file `~/.nestrc` for an example setting `firefox` as browser. Please note that
 the command `helpdesk` does not work if you have compiled NEST with MPI support,
 but you have to enter the address of the helpdesk
-(`file://$PREFIX/share/doc/nest/index.html`) manually into the browser.
-Please replace `$PREFIX` with the prefix you chose during the configuration of
-NEST. If you did not explicitly specify one, it is most likely set to `/usr` or
-`/usr/local` depending on what system you are.
+(`file:///install/path/share/doc/nest/index.html`) manually into the browser.
+Please replace `/install/path` with the path under which NEST is installed.
 
 ### Tell NEST about your MPI setup
 
 If you compiled NEST with support for distributed computing via MPI, you have
 to tell it how your`mpirun`/`mpiexec` command works by defining the function
 mpirun in your `~/.nestrc` file. This file already contains an example
-implementation that should work with [OpenMPI](http://www.openmpi.org/) library.
+implementation that should work with [OpenMPI](http://www.openmpi.org/).
 
 ## Mac OS X
 
+On the Mac, you can install NEST either via Homebrew or manually. 
+If you want to use PyNEST, you need to have a version of Python with some 
+science packages installed, see the 
+[section Python on Mac](#python-on-mac) for details. 
+
+### Installation via Homebrew
+
 The easiest way to install NEST on a Mac is to install it via the Homebrew
 package manager:
 
-1.  To install homebrew, follow the instructions here: <http://brew.sh/>
+1.  To install homebrew, follow the instructions at [brew.sh](http://brew.sh/)
 
 2.  Then, in a terminal
 
-    1.  Add the homebrew/science tap: execute 'brew tap homebrew/science'
-
+    1.  Add the homebrew/science tap by running `brew tap homebrew/science`
+    
     2.  For information on what options NEST has and what will be installed,
-        execute 'brew info nest'
-
-    3.  To install nest, execute 'brew install nest'
-
-Options have to be appended, e.g. to install NEST with PyNEST execute
-'brew install nest --with-python'.
+        run `brew info nest`
+        
+    3.  To install nest, execute `brew install nest`
 
-#Sorry, the following instructions are a little bit outdated!#
-A more detailed up-to-date instruction for installation on OSX will
-follow soon.
+Options have to be appended, e.g. to install NEST with PyNEST run
+brew install nest `--with-python`.
```

Comment:
Why are the backticks only around the `--with-python?` I think the whole line is a command, or do I misinterpret this?